### DISCUSSION TITLE
Clock 2.4

### DISCUSSION THREAD

**Discussion by BlackyHawky** [BlackyHawky/Clock#42] [Announcements]
**Fix**
 - Fix digital widgets update for Android13+;

**Improvements**
 - Digital widgets are customizable:
   - The color of all elements can be changed;
   - Font size can be changed for the main clock (only possible if no city is displayed);
   - Configure digital widgets on long press for Android 9+;
 - Some translations updated;

<hr /><em>This discussion was created from the release <a href='https://github.com/BlackyHawky/Clock/releases/tag/2.4'>Clock 2.4</a>.</em>

**Comment by NaN-Sz** [comment]
It was what i needed . great app!

**Reply by BlackyHawky** [reply]
Thank you very much for your comment. I'm glad you like it. 😉
